conf/references/mlss_conf.xml

```diff
@@ -253,55 +269,191 @@
     </collection>
 
     <collection name="hexapoda">
-      <base_collection name="shared"/>
-
-      <!-- Outgroups -->
+      <base_collection name="shared_metazoa"/>
+      <!-- outgroup -->
+      <genome name="daphnia_magna_gca020631705v2"/>
       <genome name="strigamia_maritima"/>
-      <genome name="daphnia_pulex"/>
-      <genome name="ixodes_scapularis"/>
-
-      <!-- Phthiraptera -->
-      <genome name="pediculus_humanus"/>
+      <genome name="folsomia_candida"/>
```

Comment:
could you please add a comment defining its clade (is Entognatha the right clade for this species?)


--

Reply:
Collembola I think is best. Entognatha is also correct but not accepted taxonomic clade in NCBI (missing taxon id)